scripts/package-lock.json

```diff
@@ -0,0 +1,951 @@
+{
```

Comment:
Yep, package-lock.json is a Typescript-specific thing and belongs in teh `typescript` dir